src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/MyExtensibilityPropPage.de.xlf

```diff
@@ -29,7 +29,7 @@
       </trans-unit>
       <trans-unit id="colHeaderExtensionVersion.HeaderText">
         <source>Version</source>
-        <target state="translated">Version</target>
+        <target state="translated">VERSION</target>
```

Comment:
Another capitalisation. There are more examples of capitalisation changes and white space changes throughout.